engine/jobmaster/dm/api.go

```diff
@@ -29,17 +29,17 @@ import (
 
 // TaskStatus represents status of a task
 type TaskStatus struct {
-	ExpectedStage  metadata.TaskStage
-	WorkerID       frameModel.WorkerID
-	ConfigOutdated bool
-	Status         *dmpkg.QueryStatusResponse
+	ExpectedStage  metadata.TaskStage         `json:"expected_stage"`
+	WorkerID       frameModel.WorkerID        `json:"worker_id"`
+	ConfigOutdated bool                       `json:"config_outdated"`
+	Status         *dmpkg.QueryStatusResponse `json:"status"`
 }
 
 // JobStatus represents status of a job
 type JobStatus struct {
-	JobID frameModel.MasterID
+	JobID frameModel.MasterID `json:"job_id"`
```

Comment:
Is there any regulation that we follow about using `hyphen` or `underscore` in json key.